documentation/tutorials/get-started.md

```diff
@@ -186,7 +186,7 @@ defmodule MyApp.Accounts.User do
     tokens do
       token_resource MyApp.Accounts.Token
       signing_secret fn _, _ ->
-        Application.fetch_env(:my_app, :token_signing_secret)
+        Application.fetch_env!(:my_app, :token_signing_secret)
```

Comment:
This would need to change to `{:ok, Application.fetch_env!(:my_app, :token_signing_secret)}`

I think we should keep this change as is, because `fetch_env` returns `{:ok, value} | :error`, which matches the callback signature required here.